cypress/integration/User_not_logged.js

```diff
@@ -0,0 +1,23 @@
+// @noflow
+/* global cy */
+
+describe('User not logged', () => {
```

Comment:
the description should be something like "User can search in FAQs", filename should reflect that too, as there will be more tests related to the not logged user.